src/test/java/atdd/path/web/UserAcceptanceTest.java

```diff
@@ -1,8 +1,11 @@
 package atdd.path.web;
```

Comment:
```suggestion
package atdd.web;
```

path와 별개의 패키지로 구분하면 좋을 것 같아요 :)